content/v2.0/reference/api/client-libraries/js.md

```diff
@@ -1,12 +1,187 @@
 menu:
   v2_0_ref:
     name: JavaScript
     parent: Client libraries
-    url: https://github.com/influxdata/influxdb-client-js
+v2.0/tags: [client libraries, JavaScript]
 weight: 201
 ---
+
+Use the [InfluxDB JavaScript client library](https://github.com/influxdata/influxdb-client-js) to integrate InfluxDB into JavaScript scripts and applications. This client supports both client-side (browser) and server-side (NodeJS) environments. 
+
+This guide presumes some familiarity with JavaScript, browser environments, and InfluxDB.
+If just getting started, see [Get started with InfluxDB](/v2.0/get-started/).
+
+## Before you begin
+
+1. Install Go 1.3 or later](https://golang.org/doc/install)
+
+2. Install [NodeJS](https://nodejs.org/en/download/package-manager/):
+
+3. Ensure that InfluxDB is running and you can connect to it.
+   If running InfluxDB locally, visit http://localhost:9999.
+   If using InfluxDB Cloud, visit your [InfluxDB Cloud URL](/v2.0/cloud/urls).
+
+## Easiest way to get started 
+1. Clone the [examples directory](https://github.com/influxdata/influxdb-client-js/tree/master/examples) in the [influxdb-client-js](https://github.com/influxdata/influxdb-client-js) repo. 
+2. Navigate to `examples` directory and install dependencies. 
+
+    {{< code-tabs-wrapper >}}
+    {{% code-tabs %}}
+[npm](#)
+[yarn](#)
+    {{% /code-tabs %}}
+    {{% code-tab-content %}}
+    ```sh
+# Navigate into examples directory
+cd examples 
+
+# Install dependencies
+npm install
+    ```
+    {{% /code-tab-content %}}
+    {{% code-tab-content %}}
+    ```sh
+# Navigate into examples directory
+cd examples 
+
+# Install dependencies
+yarn install
+    ```
+    {{% /code-tab-content %}}
+    {{< /code-tabs-wrapper >}}
+3. Update your `./env` and `index.html` with the name of your InfluxDB [bucket](/v2.0/organizations/buckets/), [organization](/v2.0/organizations/), [token](/v2.0/security/tokens/), and `url` which relies upon proxy to forward requests to the target InfluxDB. 
+4. Run the following command to run the application at [http://localhost:3001/examples/index.html]()
+
+    ```sh
+    npm run browser
+    ```
+
+## Boilerplate for the InfluxDB Javascript Client Lbrary  
+Use the Javascript library to write and query data to and from InfluxDB.
+
+To write a data point to InfluxDB using the JavaScript library, import the latest InfluxDB Javascript library in your script.
+
+```js
+import {InfluxDB, Point} from 'https://unpkg.com/@influxdata/influxdb-client/dist/index.browser.mjs'
+```
+
+Next, define constants for your InfluxDB [bucket](/v2.0/organizations/buckets/), [organization](/v2.0/organizations/), [token](/v2.0/security/tokens/), and `url` which  relies on a proxy to forward requests to the target InfluxDB instance. 
+
+
+```js
+const url = '/influx' 
+const token = '<my-token>'
+const org = '<my-org>'
+const bucket = '<my-bucket>'
+//variable to store the url of your local or InfluxDB Cloud instance
+const url = 'http://localhost:9999'
```

Comment:
You have two `url` constants. We could rename the first to `proxy` or this one to `influx_url`.